data_steward/analytics/cdr_ops/combined.py

```diff
@@ -0,0 +1,479 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.7.1
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# ## Notebook parameters
+
+# + tags=["parameters"]
+PROJECT_ID = ''  # identifies the project containing the datasets
+DATASET_ID = ''  # the dataset to evaluate
+BASELINE_DATASET_ID = ''  # a baseline dataset for metrics comparison
+
+# +
+from google.cloud import bigquery
+import pandas as pd
+
+from common import JINJA_ENV
+from cdr_cleaner.cleaning_rules.negative_ages import date_fields
+from utils import auth
+
+CLIENT = bigquery.Client(project=PROJECT_ID)
+
+pd.options.display.max_rows = 1000
+pd.options.display.max_colwidth = 0
+pd.options.display.max_columns = None
+pd.options.display.width = None
+
+
+def q(query):
```

Comment:
nit:  `q` is very short and not super descriptive.  Maybe expand the function name.  Objections?

Reply:
maybe `exe` or `q_exe`?

Reply:
I changed it to `execute()` 😊